pynicotine/gtkgui/ui/settings/chats.ui

```diff
@@ -309,7 +309,7 @@
             <property name="visible">True</property>
             <child>
               <object class="GtkLabel">
-                <property name="label" translatable="yes">Private chat format:</property>
+                <property name="label" translatable="yes">Private chat format (strftime):</property>
```

Comment:
These appendages don't really provide much value.

Reply:
Ideas on how to present the format? Empty text boxes are useless too, but I'd rather not provide links to random websites. The format doesn't seem to have a friendly name unfortunately.

Reply:
Links are fine, but I need stable ones. I put a link to the formatting codes in the Python documentation, but let me know if you find a better one.

Reply:
- If no value is set, then populate the text entry with the default locale value.
- Add a label beneath the entry, which presents on_entry_changed() the result of the entry format using the current time or an exemplar time such as the time the dialog was opened.

Reply:
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Reply:
I wonder if it would be easier to just change the default timestamp formats in the config. You can currently hide the timestamps by leaving the timestamp format blank, and I'm not sure if I want to change this behavior.

I think I'll just add a link for now to keep things simple. Maybe we could revisit some kind of preview in the future.